admin.py

```diff
@@ -168,9 +168,9 @@ def get(self):
 
     if settings.PROD:
       try:
-        with files.open(BIGSTORE_BUCKET + filename, 'r') as unused_f:
+        with cloudstorage.open(BIGSTORE_BUCKET + filename, 'r') as unused_f:
           pass
-      except files.file.ExistenceError, e:
+      except Exception, e:
```

Comment:
How about catching `cloudstorage.errors.Error` instead of the generic `Exception`? It looks like that's the class that all the other errors in the library extend.